app/controllers/content/index.js

```diff
@@ -82,13 +90,13 @@ export default Controller.extend(Analytics, {
         const editResubmitPreprint = 'content.project_button.edit_resubmit_preprint';
         return (
             this.get('model.provider.reviewsWorkflow') === PRE_MODERATION
-            && this.get('model.reviewsState') === REJECTED
+            && this.get('model.reviewsState') === REJECTED && this.get('isAdmin')
```

Comment:
Edit button only says resubmit if user is admin.  Needed because now all contribs can see the edit button.